roles/pulp-devel/tasks/main.yml

```diff
@@ -83,6 +109,16 @@
         virtualenv: '{{ pulp_install_dir }}'
         virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
 
+    - name: Install bindings requirements for testing locally
+      pip:
+        name: "{{ item.value.binding }}"
+        virtualenv: '{{ pulp_install_dir }}'
+        virtualenv_command: '{{ pulp_python_interpreter }} -m venv'
+      with_dict: "{{ pulp_install_plugins }}"
+      when: item.value.binding is defined
+      notify:
+        - Install pulpcore binding package via PyPI
```

Comment:
I think this needs to be done via local source

Reply:
without pulpcore binding, any plugin binding would be useless, because tests need pulpcore binding also

Reply:
Yes I just imagined it would come from a local checkout of pulpcore. Perhaps I'm thinking that is normal because I'm usually working on pulpcore and need the source install there, but other devs maybe don't.

Reply:
my idea with this is to assure the dev will have the env ready for running functional tests as soon it is provisioned, that is how I understood: https://pulp.plan.io/issues/6032#note-2

Reply:
I imagined it pbindings would be called late in the dev setup process. That would be better I think because then the bindings version received matches the version of pulpcore they are actually running. It also would be simpler because you can just let pbindings do it all.

Reply:
When I saw late in the process I mean after Pulp is running because it needs to provide the openAPI schema